src/containers/Pomodoro/Pomodoro.js

```diff
@@ -23,30 +24,81 @@ class Pomodoro extends Component {
   state = {
     timerIsRunning: false,
     seconds: 5,
-    sessionSoundStatus: Sound.status.STOPPED
+    sessionSoundStatus: Sound.status.STOPPED,
+    isBreak: false,
+    breakSoundStatus: Sound.status.STOPPED
   };
 
-  handleStartButton = () => {
-    this.setState({ timerIsRunning: true, sessionSoundStatus: Sound.status.STOPPED });
-    const timer = setInterval(() => {
+  startTimer = isBreak => {
+    if (!isBreak) {
+      this.setState({ timerIsRunning: true, isBreak });
+      const timer = setInterval(() => {
+        const { seconds } = this.state;
+        this.setState({ seconds: seconds - 1 });
+      }, 1000);
+
+      const { seconds } = this.state;
+      setTimeout(() => {
+        clearInterval(timer);
+        this.setState({
+          timerIsRunning: false,
+          seconds: seconds,
+          sessionSoundStatus: Sound.status.PLAYING,
+          isBreak: true
```

Comment:
If you do the setting of `isBreak` in both if branches, you don't need this line.